mutators.scad

```diff
@@ -64,13 +64,20 @@ module bounding_box(excess=0) {
 }
 
 
-// Module: half_of()
+// Function&Module: half_of()
 //
-// Usage:
+// Usage: as module
 //   half_of(v, [cp], [s]) ...
```

Comment:
We used to use [foo] to denote optional args in Usage, but we ran into a lot of cased where we wanted to indicate the passing of a list argument. Thus we've been moving the Usage optional args to using the syntax <foo>.